apps/client/src/index.js

```diff
@@ -8,13 +8,13 @@ import { createGlobalStyle } from "styled-components";
 import dotenv from "dotenv-flow";
 import { MatomoProvider, createInstance } from "@datapunt/matomo-tracker-react";
 import { GlobalStyle, ThemeProvider, themeColor } from "@datapunt/asc-ui";
+import "@datapunt/asc-assets/static/fonts/fonts.css";
```

Comment:
Huh? What does this have to do with refresh-button?
This has never been here? Why add it now?

Reply:
it's moved from the bottom to have our import order. 
But i think it should be set on top.

Reply:
Indeed, it was at the bottom. I will put it back and will leave it up to the eslint / prettier to deal with import order. For now resolving this